x-pack/test/security_solution_endpoint/apps/endpoint/resolver.ts

```diff
@@ -0,0 +1,197 @@
+      await testSubjects.click('resolver:graph-controls:south-button');
+
+      const NewNodeDataStyle = await pageObjects.hosts.parseStyles();
+      for (let i = 0; i < NewNodeDataStyle.length; i++) {
+        expect(parseFloat(NewNodeDataStyle[i].left)).to.lessThan(
+          parseFloat(OriginalNodeDataStyle[i].left)
+        );
+        expect(parseFloat(NewNodeDataStyle[i].top)).to.lessThan(
+          parseFloat(OriginalNodeDataStyle[i].top)
+        );
+      }
+      await (await testSubjects.find('resolver:graph-controls:center-button')).click();
+      const CenterNodeDataStyle = await pageObjects.hosts.parseStyles();
+
+      for (let i = 0; i < CenterNodeDataStyle.length; i++) {
+        expect(parseFloat(CenterNodeDataStyle[i].left)).to.equal(
+          parseFloat(OriginalNodeDataStyle[i].left)
+        );
+        expect(parseFloat(CenterNodeDataStyle[i].top)).to.equal(
+          parseFloat(OriginalNodeDataStyle[i].top)
+        );
+      }
+    });
+
+    it('resolver Nodes navigation zoom in', async () => {
+      const OriginalNodeDataStyle = await pageObjects.hosts.parseStyles();
+      await testSubjects.click('resolver:graph-controls:zoom-in');
+
+      const NewNodeDataStyle = await pageObjects.hosts.parseStyles();
+      for (let i = 1; i < NewNodeDataStyle.length; i++) {
+        expect(parseFloat(NewNodeDataStyle[i].left)).to.lessThan(
+          parseFloat(OriginalNodeDataStyle[i].left)
+        );
+        expect(parseFloat(NewNodeDataStyle[i].top)).to.lessThan(
+          parseFloat(OriginalNodeDataStyle[i].top)
+        );
+        expect(parseFloat(OriginalNodeDataStyle[i].width)).to.lessThan(
+          parseFloat(NewNodeDataStyle[i].width)
+        );
+        expect(parseFloat(OriginalNodeDataStyle[i].height)).to.lessThan(
+          parseFloat(NewNodeDataStyle[i].height)
+        );
+        await testSubjects.click('resolver:graph-controls:zoom-out');
+      }
+    });
+
+    it('resolver Nodes navigation zoom out', async () => {
+      const OriginalNodeDataStyle = await pageObjects.hosts.parseStyles();
+      await testSubjects.click('resolver:graph-controls:zoom-out');
+      const NewNodeDataStyle1 = await pageObjects.hosts.parseStyles();
+      for (let i = 1; i < OriginalNodeDataStyle.length; i++) {
+        expect(parseFloat(OriginalNodeDataStyle[i].left)).to.lessThan(
+          parseFloat(NewNodeDataStyle1[i].left)
+        );
+        expect(parseFloat(OriginalNodeDataStyle[i].top)).to.lessThan(
+          parseFloat(NewNodeDataStyle1[i].top)
+        );
+        expect(parseFloat(NewNodeDataStyle1[i].width)).to.lessThan(
+          parseFloat(OriginalNodeDataStyle[i].width)
+        );
+        expect(parseFloat(NewNodeDataStyle1[i].height)).to.lessThan(
+          parseFloat(OriginalNodeDataStyle[i].height)
+        );
+      }
+      await testSubjects.click('resolver:graph-controls:zoom-in');
+    });
+
+    after(async () => {
```

Comment:
I think the preference would be to arrange the after() block right after the before() block